web/src/components/storage/DriveEditor.test.tsx

```diff
@@ -204,7 +210,8 @@ describe("PartitionMenuItem", () => {
     const deleteRootButton = within(partitionsMenu).queryByRole("menuitem", {
```

Comment:
NP: I'd go for a `getByRole` here now that you don't have to check that such a thing is not in the document.

Reply:
Done.